frontend/test/pytest/test_jax_dynamic_api.py

```diff
@@ -347,6 +347,7 @@ def f():
         qjit(f)
 
 
+@pytest.mark.xfail
 def test_shapes_type_conversion():
```

Comment:
@dime10 @grwlf It is failing because of this PR https://github.com/google/jax/commit/a1ee8c174386124973f7d898164d487cd3e0e579 in Jax. Could let me know if you see any solution?

Reply:
As discussed in the meeting, we can change this test to verify floats _raise_ an error, although technically this is something JAX should be doing, so I think it would fine removing it altogether as well.

Reply:
Test removed